static/app/components/search/index.tsx

```diff
@@ -63,7 +62,7 @@ const defaultProps = {
   closeOnSelect: true,
 };
 
-type Props = WithRouterProps<{orgId: string}> & {
+export interface SearchProps extends WithRouterProps<{orgId: string}> {
```

Comment:
```suggestion
interface SearchProps extends WithRouterProps<{orgId: string}> {
```

Reply:
We are using the exported props in our test